app/views/common_parameters/_parameter.html.erb

```diff
@@ -3,15 +3,15 @@
   <table class="row">
     <tbody>
       <tr class="form-group <%= 'has-error' if f.object.errors.any? %>">
-        <% if controller_name =~ /(host|hostgroup)/ %>
```

Comment:
I think I see what you tried to fix here, with the number of columns not being right, it's just it has a large indent now on the left hand side of a domain parameter tab...